src/pages/new/volunteer/index.js

```diff
@@ -35,6 +36,7 @@ export default () => (
       title="Volunteering during COVID-19"
       description="Our programs have shifted to remote alternatives. While our means of coming together are different, our research and Datathon events will be conducted online. Simply put, our goal is still the same: to connect this community with resources that can help them address their challenges, become permanently housed, and on track to live happy and productive lives."
     />
+    <Spacer heightDesktop="120px" heightMobile="0" />
```

Comment:
Shouldn't there be 100px of spacing in desktop here and 80px of spacing in mobile?

Reply:
ok